landoapi/app.py

```diff
@@ -24,9 +29,17 @@ def create_app(version_path):
 
     # Get the Flask app being wrapped by the Connexion app.
     flask_app = app.app
+
     flask_app.config['VERSION_PATH'] = version_path
     log_config_change('VERSION_PATH', version_path)
 
+    version_info = json.load(open(version_path))
+    logger.info(version_info, 'app.version')
+
+    this_app_version = version_info['commit']
+    environment = version_info['build']
```

Comment:
"environment label a host lives under"... ?

You mean like "this is running in dev/stage/prod"? If so, that's available from the env as the ENV var.